android/src/main/java/com/swmansion/rnscreens/Screen.kt

```diff
@@ -80,7 +80,7 @@ class Screen(context: ReactContext?) : FabricEnabledViewGroup(context) {
     private fun updateScreenSizePaper(width: Int, height: Int) {
         val reactContext = context as ReactContext
         reactContext.runOnNativeModulesQueueThread(
-            object : GuardedRunnable(reactContext) {
+            object : GuardedRunnable(reactContext.exceptionHandler) {
```

Comment:
I saw the warning you pasted in PR description, but I checked in [source code](https://github.com/facebook/react-native/blob/362abb9ffea5af7e6e5616849fbce28166c758a0/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/GuardedRunnable.java#L18-L24) & this is not deprecated 🤷🏻‍♂️  

It's fine I guess

Reply:
That's actually quite weird - according to the source code from Android Studio, GuardedRunnable with constructor with `ReactContext` parameter is deprecated though - I'm concerned why there's any difference here 🤔 

<img width="560" alt="image" src="https://github.com/software-mansion/react-native-screens/assets/23281839/35adb871-efa0-43be-8ae4-dea6ffaf1d0a">

Reply:
I've found similar fix here: https://github.com/software-mansion/react-native-reanimated/pull/1031

Reply:
The reason why it is not deprecated right now is that since 0.73 this constructor has been de-deprecated 😄 
I can see on [0.72](https://github.com/facebook/react-native/blob/0.72-stable/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/GuardedRunnable.java) that `@Deprecated` is still there and has been removed since [this](https://github.com/facebook/react-native/commit/330639f74018cfeda28e74c6dae8110d97603860) commit.